Makefile

```
@@ -224,22 +271,46 @@ update: ## Update node modules to latest allowed versions — always runs locall
+else
 	$(DOCKER_COMPOSE) down --remove-orphans
+endif
 
 sh: ## Log to the docker container
+ifeq ($(DIND), 1)
+	$(DOCKER_COMPOSE) $(COMPOSE_FILES) exec dev sh
+else
 	$(DOCKER_COMPOSE) exec dev sh
+endif
 
 ps: ## Log to the docker container
```

checkmake (0.2.2)

[warning] Target "ps" should be declared PHONY.

(phonydeclared)